modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/ContentionClosure.java

```diff
@@ -105,6 +105,9 @@ public ContentionClosure(int minQueueSize, int maxPrint) {
                         catch (GridCacheEntryRemovedException ignored) {
                             cached = entry.context().cache().entryEx(entry.key());
                         }
+                        finally {
+                            cached.touch();
```

Comment:
It seems I was wrong and touch is not needed here, because it could trigger various exceptions on tx prepare/lock acquisition. Touch will be called by code handling the specific transaction. Please remove.